extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/resources/translations/es.json

```diff
@@ -72,34 +72,42 @@
     },
 
     "HOME" : {
-        "INFO_SHARED_BY" : "Compartidas con {USERNAME}"
+        "INFO_SHARED_BY" : "Compartidas por {USERNAME}"
     },
 
     "PASSWORD_POLICY" : {
 
-        "ERROR_CONTAINS_USERNAME"      : "Las contraseñas no deberían contener el nombre de usuario.",
+        "ERROR_CONTAINS_USERNAME"      : "Las contraseñas no pueden contener el nombre de usuario.",
         "ERROR_REQUIRES_DIGIT"         : "Las contraseñas deben contener al menos un número.",
         "ERROR_REQUIRES_MULTIPLE_CASE" : "Las contraseñas deben contener caractéres en mayúscula y minúscula.",
         "ERROR_REQUIRES_NON_ALNUM"     : "Las contraseñas deben contener al menos un símbolo.",
-        "ERROR_REUSED"                 : "Esta contraseña ya se ha usado. Por favor no vuelva a usar ninguna de las  {HISTORY_SIZE} {HISTORY_SIZE, plural, one{password} other{passwords}} anteriores.",
-        "ERROR_TOO_SHORT"              : "Las contraseñas deben ser al menos {LENGTH} {LENGTH, plural, one{character} other{characters}} de largo.",
-        "ERROR_TOO_YOUNG"              : "La contraseña de esta cuenta ya ha sido renovada. Por favor espere al menos {WAIT} mas {WAIT, plural, one{day} other{days}} antes de cambiarla de nuevo."
+        "ERROR_REUSED"                 : "Esta contraseña ya se ha utilizado. Por favor, no vuelva a usar ninguna de las anteriores {HISTORY_SIZE} {HISTORY_SIZE, plural, one{password} other{passwords}}.",
+        "ERROR_TOO_SHORT"              : "Las contraseñas deben ser como mínimo {LENGTH} {LENGTH, plural, one{character} other{characters}} de largo.",
+        "ERROR_TOO_YOUNG"              : "La contraseña de esta cuenta ya se ha cambiado. Espere como mínimo {WAIT} mas {WAIT, plural, one{day} other{days}} antes de volverla a cambiar."
```

Comment:
The pluralization rules in these translations will say "0 days", "1 day", "2 days", etc., with "day" and "days" being in English. Is this correct? (Same for "character" and "password").